domains/marcus.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "A personal website",
+  "repo": "https://github.com/HeroGamers/HeroGamers",
+  "owner": {
+    "username": "HeroGamers",
+    "email": "hero@herogamers.xyz",
+    "twitter": "ItsHeroGamers"
+  },
+  "record": {
+    "CNAME": "herogamers.github.io/HeroGamers"
```

Comment:
CNAME records cannot be a full url. It has to be a host name. I'd suggest you rename your repo to `herogamers.github.io`.

You can take a look at the [github pages guide](https://guides.github.com/features/pages/).

Reply:
Change the github pages settings some time after this pull request is merged. It will take some time for the dns changes to propogate through the network.

Reply:
According to [this GitHub doc](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site), changing the CNAME to be `herogamers.github.io`, will also work with the custom repository that I'm linking to, sorry for the confusion and for putting a URL instead of a hostname there.

Reply:
Please undo any settings changes you made. Let the pr get merged, wait till `domain.is-a.dev` shows you github's 404 page. Then add `domain.is-a.dev` in your repo settings and check the 'Ensure HTTPS' button.